go/engine/login_provision.go

```diff
@@ -1192,7 +1192,7 @@ func (e *loginProvision) cleanup(m libkb.MetaContext) {
 
 	// the best way to cleanup is to logout...
 	m.Debug("an error occurred during provisioning, logging out")
-	m.G().Logout(m.Ctx())
```

Comment:
there are 3 more of these to delete

Reply:
Only 1 more with _fatal consequences_, the remaining calls are actually somewhat required (and hard to rework) but they are fixed with the restoreSession fn.